move HYGROMETER_SENSOR from development.xml to common.xml

gitfishup opened the pull request:
Based on the new message, I implemented the display of the hygrometer information on the ground station.
NOTE:
      I move HYGROMETER_SENSOR from development.xml to common.xml to test. Because Mission Planner and ardupilot don't have the development.xml file. So , ardupilot , MissionPlanner , PX4-Autopilot , Qgroundcontroll are use the  HYGROMETER_SENSOR message from common.xml.
      
Here，i will show you the display on QGC and MissionPlanner：
![bd311c6af9f0bf0fe303c4e090c7b6a](https://user-images.githubusercontent.com/69180601/129299071-af30f0d9-5f6c-403b-83ca-29afe5a3ff94.png)
![1d86cf6814e4c35d9c48e91b3a53414](https://user-images.githubusercontent.com/69180601/129299074-a616a9c5-4b5b-4642-997b-6aba712c99f0.png)
And here is the videos  show that the dynamic change of hygrometer message(The airspeed is heating)




https://user-images.githubusercontent.com/69180601/129299677-2b9b0533-860e-42f5-9239-96bafb46f90f.mp4


https://user-images.githubusercontent.com/69180601/129299686-8b70d95a-ddbf-4be8-ac32-35a1f6b4c08d.mp4



@hamishwillee if you need more details about test , i wil show you .

gitfishup commented:
@hamishwillee Could you please have review?

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -6893,5 +6893,13 @@
+      <description>temperature and humidity from hygrometer.</description>
+      <field type="uint8_t" name="id" instance="true">Hygrometer ID</field>
+      <field type="int16_t" name="temperature" units="cdegC">temperature</field>
+      <field type="uint16_t" name="humidity" units="c%">humidity</field>
```
Please capitalise the description from `humidity` to `Humidity`

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -6893,5 +6893,13 @@
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>temperature and humidity from hygrometer.</description>
+      <field type="uint8_t" name="id" instance="true">Hygrometer ID</field>
+      <field type="int16_t" name="temperature" units="cdegC">temperature</field>
```
Please capitalise the description from `temperature` to `Temperature`

hamishwillee commented on message_definitions/v1.0/common.xml:
```diff
@@ -6893,5 +6893,13 @@
       <field type="uint8_t[250]" name="messages">Concatenation of encoded OpenDroneID messages. Shall be filled with nulls in the unused portion of the field.</field>
     </message>
+    <message id="12920" name="HYGROMETER_SENSOR">
+      <wip/>
```
Please remove this line and the next line (i.e. if you're moving it into common.xml is is no longer "WIP". You won't be able to change it easily after this. That is why pre-testing is important.

hamishwillee commented:
Hi @gitfishup 

Thanks very much. I've added some minor comments and am nearly ready to merge. 

However this is failing CI tests. Are you able to run the formatter. Usually I do this on Ubuntu linux:

`./scripts/format_xml.sh ./message_definitions/v1.0/development.xml`

gitfishup commented:
> Next time we go through through this process, logs are also useful. They show us things like message rates etc.

@hamishwillee Thanks! Do you mean the flight controller logs?

gitfishup commented:
> Hi @gitfishup
> 
> Thanks very much. I've added some minor comments and am nearly ready to merge.
> 
> However this is failing CI tests. Are you able to run the formatter. Usually I do this on Ubuntu linux:
> 
> `./scripts/format_xml.sh ./message_definitions/v1.0/development.xml`
> 
> P.S. Next time we go through through this process, logs are also useful. They show us things like message rates etc.

@hamishwillee I have changed them. And i will upload the logs here soon.

hamishwillee commented:
Thanks. Merging :-)

gitfishup commented:
> Thanks. Merging :-)

I'm so sorry about typos. I didn't fix everything.

hamishwillee commented:
No worries!
